Marlin/pins.h

```diff
@@ -135,7 +137,7 @@
   #include "pins_MEGATRONICS_3.h"
 #elif MB(MEGATRONICS_31)
   #define MEGATRONICS_31
-  #include "pins_MEGATRONICS_3.h"
+  #include "pins_MEGATRONICS_3.h" 
```

Comment:
Whoops! Are you familiar with rebase/squash? After you remove this extraneous space, it would be good to still have only 1 commit.